catroid/src/createAtSchool/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -40,8 +41,10 @@
 	public static final File DEFAULT_ROOT_DIRECTORY = CatroidApplication.getAppContext().getFilesDir();
 
 	public static final File EXTERNAL_STORAGE_ROOT_DIRECTORY = new File(
-			Environment.getExternalStorageDirectory().getAbsolutePath(), POCKET_CODE_EXTERNAL_STORAGE_FOLDER_NAME);
+			Environment.getExternalStorageDirectory(), POCKET_CODE_EXTERNAL_STORAGE_FOLDER_NAME);
 
+	public static final File EXTERNAL_STORAGE_ROOT_EXPORT_DIRECTORY = new File(
```

Comment:
why is this in FlavoredConstants and not in Constants?

Reply:
I didn't thought that much where it belongs, because the EXTERNAL_STORAGE_ROOT_DIRECTORY is also in the FlavouredConstants. Should I move both to Constants?